addons/mrp/tests/test_consume_tracked_component.py

```diff
@@ -0,0 +1,161 @@
+from odoo.tests import tagged
+from odoo.addons.mrp.tests.common_consume_tracked_component import TestConsumeTrackedComponentCommon
+
+@tagged('post_install', '-at_install')
+class TestConsumeTrackedComponent(TestConsumeTrackedComponentCommon):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
```

Comment:
This isn't necessary since you're not adding anything to or changing anything in the class
```suggestion

```